src/components/ChallengePane/ChallengePane.jsx

```diff
@@ -118,6 +127,7 @@ export class ChallengePane extends Component {
 
   render() {
     const showingArchived = this.props.history.location.search.includes("archived=true");
+    const filteringGlobal = this.props.history.location.search.includes("filterGlobal=true")
```

Comment:
`filterGlobal=true` is a double negative which makes it a bit confusing to reason about. I think you should rename it to `global`. When `global=true`, global challenges should be included (similar to how archived challenges are included when `archived=true`) and when `global=false`, global challenges should be excluded.